core/src/core-api/i18n.js

```diff
@@ -30,7 +30,7 @@ class LuigiI18NManager {
 
   /**
    * Gets the current locale.
-   * @returns {string} current locale
+   * @returns {string}: current locale
```

Comment:
Type definitions cannot be followed by a colon. If you run `lerna run docu` you see that it is broken. Please change all occurrences in this file.